src/test/groovy/org/prebid/server/functional/model/request/auction/ImpExt.groovy

```diff
@@ -20,6 +20,7 @@ class ImpExt {
     ImpExtContextData data
     String tid
     String gpid
+    Integer ae
```

Comment:
It's only OpenX specific? Or this param applies to PBS Core?

Reply:
Currently work only for OpenX but it's core